package.json

```diff
@@ -39,46 +39,43 @@
     "plugin"
   ],
   "dependencies": {
-    "@typescript-eslint/eslint-plugin": "3.5.0",
-    "@typescript-eslint/parser": "3.5.0",
+    "@typescript-eslint/eslint-plugin": "3.8.0",
+    "@typescript-eslint/parser": "3.8.0",
     "babel-eslint": "10.1.0",
     "eslint-config-prettier": "6.11.0",
     "eslint-plugin-babel": "5.3.1",
     "eslint-plugin-import": "2.22.0",
-    "eslint-plugin-jest": "23.17.1",
+    "eslint-plugin-jest": "23.20.0",
     "eslint-plugin-jsx-a11y": "6.3.1",
     "eslint-plugin-node": "11.1.0",
     "eslint-plugin-prettier": "3.1.4",
     "eslint-plugin-promise": "4.2.1",
-    "eslint-plugin-react": "7.20.2",
-    "eslint-plugin-react-hooks": "4.0.5",
-    "eslint-plugin-unicorn": "20.1.0"
+    "eslint-plugin-react": "7.20.5",
+    "eslint-plugin-react-hooks": "4.0.8",
+    "eslint-plugin-unicorn": "21.0.0"
   },
   "devDependencies": {
-    "eslint": "5.16.0",
+    "eslint": "7.6.0",
     "eslint-plugin-self": "1.2.1",
     "husky": "4.2.5",
     "lint-staged": "10.2.11",
     "npm-run-all": "4.1.5",
     "prettier": "2.0.5",
-    "sortpack": "2.1.5"
+    "sortpack": "2.1.6"
   },
   "engines": {
     "node": ">=8.0.0"
   },
   "lint-staged": {
     "*.js": [
-      "prettier --write",
-      "sortpack",
-      "git add"
```

Comment:
call `git add` isn't necessary anymore and calling sortpack when only js files have been changed seemed unnecessary

Reply:
I see https://github.com/okonet/lint-staged#migration